new-content/learning-resources/textbook-1-3.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Qiskit Textbook Chapters 1 - 3
+to: https://qiskit.org/textbook
```

Comment:
This link points to the main page of textbook, not to the first chapter. We already have a link to textbook, makes sense to have two cards linking to the same page?

Reply:
Good catch. Thank you!